modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/JaxRSServerCodegen.java

```diff
@@ -118,17 +118,22 @@ public void addOperationToGroup(String tag, String resourcePath, Operation opera
         co.baseName = basePath;
     }
 
+    @Override
     public Map<String, Object> postProcessOperations(Map<String, Object> objs) {
         Map<String, Object> operations = (Map<String, Object>) objs.get("operations");
         if (operations != null) {
             List<CodegenOperation> ops = (List<CodegenOperation>) operations.get("operation");
             for (CodegenOperation operation : ops) {
                 List<CodegenResponse> responses = operation.responses;
+                operation.summary = escapeText(operation.summary.trim());
```

Comment:
This `.trim()` breaks when an operation has no summary, like in this example:

```
swagger: '2.0'
info:
  title: Example API
  description:
      bla bla
      
      and one more line.
basePath: /api
paths:
  /example-path:
    put:
      responses:
        201:
          description: |
            Example was created.

            No content.
```

(The same problem happens some lines down when one response has no message.)

Can we put the `.trim()` inside the `escapeText()` method?

`escapeText()` is called from those places (all in DefaultCodegen), in addition to these here:

* fromOperation for the summary and notes (Oops, this means it is actually not needed in this line ... that's what I get from trying to solve multiple problems in one step, whithout verifiying that they actually are problems)
* fromModel (for the description)
* fromParameter (for the description)
* fromProperty (for the description)

I guess I could also add it into DefaultCodegen.fromResponse (for the description, again).
And from my judgement, in all of these places leading or trailing whitespace are not significant, and can be removed before escaping.